README.md

```diff
@@ -2,6 +2,30 @@
 
 Click here to view Live [Magic-books](https://tanzila-magic-books.herokuapp.com/).
```

Comment:
This section of your README  should include a description of your Magic Book project. the live link is not yet updated to your current app version and not be added here. kindly fix it

Reply:
Hello 

I rectified mistakes on the readme file and updated the old link with the recent working link.